app/views/new_document/choose_document_type.html.erb

```diff
@@ -12,7 +12,10 @@
             value: document_type.id,
             text: document_type.label,
             hint_text: document_type.description,
-            data_attributes: { gtm: "choose-document-type" },
+            data_attributes: {
+              gtm: "choose-document-type",
+              "gtm-action": document_type.label,
```

Comment:
by the way you can do these as `gtm_action` and they'll be converted to `data-gtm-action` attributes

Reply:
I wondered about that, but I recall we do it this way elsewhere. I think I prefer this way, as it's clearer what the output will be to me.